apps/api/src/app/content-templates/usecases/compile-template/email/compile-email-template.usecase.ts

```diff
@@ -49,12 +38,25 @@ export class CompileEmailTemplateUsecase {
         useNovuDefault = true;
       }
     }
+    const layoutVariables = layout?.variables || [];
+    const defaultPayload = verifyPayloadService.verifyPayload(layoutVariables, command.payload);
```

Comment:
So defaultValue defined for the layout variables will also apply when sending email/preview/test